api/app/services/repository_sync_service.rb

```diff
@@ -7,12 +7,12 @@ def index
     gitland_repository = Gitland::Repository.new(@repository)
     gitland_repository.pull
 
-    @repository.transaction do
```

Comment:
I removed the transaction because I observed that sometimes it time out. I'm not sure why, but i'm sure the team next year can figure it out :)